### PR TITLE
Cleanup SA4021

### PR DESCRIPTION
**Pull request by gliptak**
Signed-off-by: Gábor Lipták <gliptak@gmail.com>

https://staticcheck.io/docs/checks#SA4021

### CODE REVIEW

File: steampipeconfig/modconfig/mod_resource_tree.go [hunk 1]
```diff
@@ -59,7 +59,7 @@ func (m *Mod) addResourcesIntoTree(sourceMod *Mod) error {
 }
 
 func (m *Mod) addItemIntoResourceTree(item ModTreeItem) error {
```
Comment (kaidaguerre) [inline steampipeconfig/modconfig/mod_resource_tree.go:61]: this function could be tidied to 
```
func (m *Mod) addItemIntoResourceTree(item ModTreeItem) error {
	for _, p := range m.getParents(item) {
		item.AddParent(p)
		p.AddChild(item)
	}
	return nil
}
```

**Review by kaidaguerre — CHANGES_REQUESTED** [review]
I would go a little further with the tidying of `addItemIntoResourceTree` - see comment

**Comment by kaidaguerre** [comment]
many thanks for the contribution @gliptak - one further change suggested

**Comment by gliptak** [comment]
@kaidaguerre updated

**Pull request merged**